src/shared/index.ts

```diff
@@ -126,6 +126,9 @@ export const getDownloadersForUri = (uri: string) => {
     ];
   }
 
+  if (uri.startsWith("https://"))
```

Comment:
Should probably support unencrypted as well.
```suggestion
  if (uri.startsWith("http://") || uri.startsWith("https://"))
```